spring-cloud-dataflow-docs/src/main/asciidoc/dashboard.adoc

```diff
@@ -0,0 +1,212 @@
+[[dashboard]]
+= Dashboard
+
+[partintro]
+--
+This section describe how to use the Dashboard of Spring Cloud Data Flow.
+--
+
+[[dashboard-introduction]]
+== Introduction
+
+Spring Cloud Data Flow provides a browser-based GUI which currently has 6 sections:
+
+* **Apps** Lists all available applications and provides the control to register/unregister them
+* **Runtime** Provides the Dataflow cluster view with the list of all running applications
+* **Streams** Deploy/undeploy Stream Definitions
+* **Tasks** List, create, launch and destroy Task Definitions
+* **Jobs** Perform Batch Job related tasks
```

Comment:
I would use "Job related *functions*" instead of *tasks*.  Just to eliminate confusion.